cli/src/test/java/com/devonfw/tools/ide/environment/EnvironmentVariablesPropertiesFileTest.java

```diff
@@ -37,4 +42,95 @@ public void testLoad() {
     assertThat(variables.getVariables()).hasSize(4);
   }
 
+  @Test
+  void testSave() {
+
+    // arrange
+    List<String> linesToWrite = new ArrayList<>();
+    linesToWrite.add("# first comment");
+    linesToWrite.add("# second comment");
+    linesToWrite.add("var0=0");
+    linesToWrite.add("var1=1");
+    linesToWrite.add("var2=2");
+    linesToWrite.add("export var3=3");
+    linesToWrite.add("export var4=4");
+    linesToWrite.add("export var5=5");
+    linesToWrite.add("export var6=6");
+    linesToWrite.add("# third comment");
+    linesToWrite.add("var7=7");
+    linesToWrite.add("var8=8");
+    linesToWrite.add("# 4th comment");
+    linesToWrite.add("# 5th comment");
+    linesToWrite.add("var9=9");
+
+    Path propertiesFilePath = Path.of("target/tmp-EnvironmentVariablesPropertiesFileTest-ide.properties");
+    try {
+      Files.write(propertiesFilePath, linesToWrite, StandardOpenOption.CREATE_NEW);
+    } catch (IOException e) {
+      System.err.println("File already exists or an error occurred: " + e.getMessage());
+    }
+    // check if this writing was correct
+    try {
+      List<String> lines = Files.readAllLines(propertiesFilePath);
+      for (int i = 0; i < lines.size(); i++) {
+        assertThat(lines.get(i)).isEqualTo(linesToWrite.get(i));
```

Comment:
actually you are not asserting our code here but the fundamentals of Java.
While this is OK, it feels a little schizophrenic to write tests that do not trust that Java is working...

Reply:
yep, just wanted to make sure the file is exactly like I want it to be:)